app/controllers/api/cookies_consent_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Api
+  class CookiesConsentController < BaseController
+    respond_to :json
+
+    def initialize
+      super
+      @cookie_name = 'cookies_consent'
+    end
```

Comment:
I wouldn't override the constructor to avoid coupling with Rails internals. Keep in mind it belongs to the framework and it might change in future versions and receive any kind of low-level optimizations that could break this controller.

Also, I think what you're trying to achieve is better suited as a constant. Like

```ruby
COOKIE_NAME = 'cookies_consent'.freeze
```

Reply:
Yes, thanks. Done. #learning_ruby